integration/test_legalhold.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+from datetime import datetime
+
+user_uid = 977335752891390447
+policy_uid = 894163190167359975
+timestamp = str(int(datetime.now().timestamp()))
+matter_name = "integration test matter {}".format(timestamp)
+policy_name = "integration test policy {}".format(timestamp)
+
+
+@pytest.fixture(scope='module')
+def test_create_policy(connection):
```

Comment:
Should this be renamed to `legalholduid`? Needed this to make sure a new policy is created for every test execution. 

Had to create it as a fixture to get the new id in other methods. However, pytest doesn't consider a fixture as a test.